app/src/main/java/dev/msfjarvis/aps/util/settings/Migrations.kt

```diff
@@ -132,3 +133,18 @@ private fun migrateToDiceware(sharedPrefs: SharedPreferences) {
     }
   }
 }
+
+private fun removeExternalStorageProperties(prefs: SharedPreferences) {
+  logcat(TAG, INFO) { "Removing preferences related to external storage" }
+  prefs.edit {
+    if (prefs.contains(PreferenceKeys.GIT_EXTERNAL)) {
+      if (prefs.getBoolean(PreferenceKeys.GIT_EXTERNAL, false)) {
+        putBoolean(PreferenceKeys.GIT_EXTERNAL_MIGRATED, true)
+      }
+      remove(PreferenceKeys.GIT_EXTERNAL)
+    }
+    if (prefs.contains(PreferenceKeys.GIT_EXTERNAL_REPO)) {
+      remove(PreferenceKeys.GIT_EXTERNAL_REPO)
+    }
+  }
+}
```

Comment:
Should we allow people to sync their already existing stores before removing the functionality if they are at an android version lower than 12? 🤔 

Another thing is how would the app work after this is removed? Do we need to reinitialize the store?

Reply:
The store's not going anywhere, they can sync it themselves with whatever tools they prefer, or they can just downgrade. Removing the preferences forces the app to the onboarding screen, which works for our needs.